app/main/views/g_cloud.py

```diff
@@ -33,8 +33,11 @@
 from app import search_api_client, data_api_client, content_loader
 
 
+PROJECT_END_SEARCH_LIMIT = 100
```

Comment:
Can you stick a comment in here that really we should be getting the limit from the API (somehow). Don't propose we do that now, but some kind of note to make sure we keep the two in sync is important for maintainability.